client/my-sites/themes/themes-selection.jsx

```diff
@@ -68,7 +68,7 @@ class ThemesSelection extends Component {
 		}
 	}
 
-	recordSearchResultsClick = ( themeId, resultsRank, action ) => {
+	recordSearchResultsClick = ( themeId, resultsRank, action, variation = '@theme' ) => {
```

Comment:
It's unfortunate that we use `@theme` in events such as `wpcom_site_launched` but `default` for the onboarding design picker events. I went with `@theme` here, but open to suggestions 🙂